http/README.md

```diff
@@ -0,0 +1,114 @@
+# HTTP
+
+| Lifecycle Stage | Maturity      | Status | Latest Revision |
+| --------------- | ------------- | ------ | --------------- |
+| 1A              | Working Draft | Active | r0, 2023-01-23  |
+
+Authors: [@marten-seemann, @MarcoPolo]
+
+Interest Group: [@lidel]
+
+[@marten-seemann]: https://github.com/marten-seemann
+[@MarcoPolo]: https://github.com/MarcoPolo
+[@lidel]: https://github.com/lidel
+
+## Introduction
+
+This document defines how libp2p nodes can offer and use an HTTP transport alongside their other transports to support application protocols with HTTP semantics. This allows a wider variety of nodes to participate in the libp2p network, for example:
+
+- Browsers communicating with other libp2p nodes without needing a WebSocket, WebTransport, or WebRTC connection.
+- HTTP only edge workers can run application protocols and respond to peers on the network.
+- `curl` from the command line can make requests to other libp2p nodes.
+
+As well as allowing application protocols to make use of HTTP intermediaries such as HTTP caching and layer 7 proxying and load balancing. This is all in addition to the existing features that libp2p provides such as:
+
+- Connectivity – Work on top of WebRTC, WebTransport, QUIC, TCP, or an HTTP transport.
+- Hole punching – Work with peers behind NATs.
+- Peer ID Authentication – Authenticate your peer by their libp2p peer id.
+- Peer discovery – Learn about a peer given their peer id.
+
+## HTTP Transport vs HTTP Semantics
```

Comment:
I don't know if there is precedent for this in libp2p specs, but I thought the visual diagram you had created for earlier HTTP discussions was useful.

Reply:
Hesitant to include that diagram since it's a png and hard to update. I've tried replicating with Mermaid so that it's text based. wdyt? I don't think it's a nice and maybe a bit more confusing than the text.

Reply:
I've attached the image I was referring to so others are aware.

It seems like that diagram was particularly useful when it came to talking about how "vanilla/simple request response" maps.  That's not in scope for this spec.  I'm still not opposed to having the diagram you have, although I do think it would be useful to color code the lines like you did in the diagram below to cover the "HTTP over libp2p" and "Just HTTP" cases.

![Libp2p request response](https://github.com/libp2p/specs/assets/85411/dd5a8718-7d7e-4032-b045-faba0d0c6c80)